<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectTab</class>
 <widget class="QWidget" name="ProjectTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1416</width>
    <height>894</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_MC">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>34</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Markov Chain</string>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameInput">
        <layout class="QVBoxLayout" name="layoutInput_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QTabWidget" name="tabWidget_inputVisualisation">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="mc1">
            <attribute name="title">
             <string>MC 1</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QScrollArea" name="scrollArea">
                  <property name="styleSheet">
                   <string notr="true">background-color: white</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <property name="verticalScrollBarPolicy">
                   <enum>Qt::ScrollBarAlwaysOn</enum>
                  </property>
                  <property name="horizontalScrollBarPolicy">
                   <enum>Qt::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="wid_scrollArea">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>577</width>
                     <height>695</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <widget class="QWidget" name="wid_initialStateVectorTitle" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>70</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: white</string>
                      </property>
                      <widget class="QLabel" name="la_initialStateVector">
                       <property name="geometry">
                        <rect>
                         <x>20</x>
                         <y>10</y>
                         <width>571</width>
                         <height>60</height>
                        </rect>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="font">
                        <font>
                         <pointsize>14</pointsize>
                         <weight>75</weight>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="text">
                        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:400;&quot;&gt;Initial state distribution:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                       </property>
                       <property name="scaledContents">
                        <bool>false</bool>
                       </property>
                      </widget>
                      <widget class="QWidget" name="wid_initialStateVector_2" native="true">
                       <property name="geometry">
                        <rect>
                         <x>0</x>
                         <y>77</y>
                         <width>555</width>
                         <height>70</height>
                        </rect>
                       </property>
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>70</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">background-color: white</string>
                       </property>
                      </widget>
                     </widget>
                    </item>
                    <item>
                     <widget class="QWidget" name="wid_initialStateVector" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>70</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: white</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QWidget" name="wid_factorDistriution" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>70</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: white</string>
                      </property>
                      <widget class="QLabel" name="la_factorDistribution">
                       <property name="geometry">
                        <rect>
                         <x>20</x>
                         <y>10</y>
                         <width>571</width>
                         <height>60</height>
                        </rect>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="font">
                        <font>
                         <pointsize>14</pointsize>
                         <weight>75</weight>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="text">
                        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:400;&quot;&gt;Factor distribution:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                       </property>
                       <property name="scaledContents">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </widget>
                    </item>
                    <item>
                     <widget class="QWidget" name="wid_factorDistribution" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>70</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: white</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="layoutInput"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QWidget" name="widget" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>11</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_addMC">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>Add Markov Chain</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>13</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_MC_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>144</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>34</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Results</string>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_resultVisualisation">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="w_calculation_characteristivs" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_chMethod">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Analyse Markov Chain</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_cancelAnalysis">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Cancel Analysis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_currentPrecision">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="la_currenPrecision">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Current precision:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_currentStep">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="la_currentStep">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Current step:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
